frontend/src/services/createPostService.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/prefer-default-export
+export const submitPost = async postInfo => {
+  const parameters = {
+    ...postInfo,
+    sender_created_at: new Date().toISOString(),
+  };
+  return fetch('/api/post/create', { method: 'POST', body: parameters });
```

Comment:
Should we return the result of `fetch` directly here? It might be better to marshal the result into a nice format that we can use directly and raise an error when the server returns a bad status.

For example (haven't tested it so might not work 100%):
```javascript
const response = await fetch('/api/post/create', { method: 'POST', body: parameters });
if(response.ok) {
    return await response.json();
} else {
    throw new Error(await response.json());
}
```

Reply:
I like your idea. The `CreatePostContainer` shouldn't need to handle/be aware of an fetch `Response` object directly.